final_task/setup.py

```diff
@@ -0,0 +1,18 @@
+import sys
+sys.path.append('rss_reader')
+from setuptools import setup, find_packages
+from rss_reader import rss_reader
+
+setup(
+    name='rss_reader',
+    version=rss_reader.VERSION,
```

Comment:
In this specific case it is better either hardcode this value or import in from some file.